apps/pre-award/lambdas/application-deadline-reminder/data.py

```diff
@@ -27,7 +27,21 @@ def get_data(endpoint, params: Optional[dict] = None):
         logging.error("There was a problem retrieving response from"
         f" {endpoint}. Status code: {response.status_code}")
         return None
+    
+def get_data_safe(endpoint, params: Optional[dict] = None):
+    try:
+        response = requests.get(endpoint, params=params)
+        response.raise_for_status()
+        return response.json()
+    except:
```

Comment:
Can we handle generic exception here HttpError.

Reply:
Sure, updated.